packages/strapi-design-system/src/Select/SelectButton.js

```diff
@@ -0,0 +1,70 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { KeyboardKeys } from '../helpers/keyboardKeys';
+
+const StyledSelectButton = styled.button`
+  text-align: left;
+  border: none;
+  padding-left: ${({ theme, hasLeftAction }) => (hasLeftAction ? 0 : theme.spaces[4])};
+  padding-right: ${({ theme, hasRightAction }) => (hasRightAction ? 0 : theme.spaces[4])};
+  padding-top: ${({ theme }) => `${theme.spaces[3]}`};
+  padding-bottom: ${({ theme }) => `${theme.spaces[3]}`};
+  display: block;
+  width: 100%;
+  background: transparent;
+
+  // The focus state is moved to the parent thanks to :focus-within
+  &:focus {
+    outline: none;
+  }
+`;
+
+export const SelectButton = forwardRef(({ children, labelledBy, expanded, onTrigger, disabled, ...props }, ref) => {
+  const handleKeyDown = (e) => {
+    if (disabled) return;
+
+    switch (e.key) {
```

Comment:
Can you add a default case as well, and add the linter rule?

Reply:
yes sure 😊